backend/inspirehep/matcher/config.py

```diff
@@ -5,6 +5,9 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
+from inspirehep.config import SEARCH_INDEX_PREFIX
```

Comment:
This will take take the hardcoded value of the prefix and will not take in consideration the env changes or the overrides (i.e. ``current_app``)